lwallet/Pos/Wallet/Light/Mode.hs

```diff
@@ -44,37 +49,48 @@ import           Pos.Wallet.Light.State.Core      (gsAdoptedBVDataWallet)
 import           Pos.Wallet.WalletMode            (MonadBlockchainInfo (..),
                                                    MonadUpdates (..))
 
-modeContext [d|
-    data LightWalletContext = LightWalletContext
-        !(PeerStateTag     ::: PeerStateCtx Production)
-        !(KeyData          ::: KeyData)
-        !(WalletState      ::: WalletState)
-        !(ReportingContext ::: ReportingContext)
-        !(DiscoveryTag     ::: Set NodeId)
-        !(JsonLogConfig    ::: JsonLogConfig)
-        !(LoggerName       ::: LoggerName)
-    |]
+type LightWalletSscType = SscGodTossing
+-- type LightWalletSscType = SscNistBeacon
```

Comment:
No, it's just a showcase how it could be done alternatively. This is how it's done for `WalletSscType`:

```
-- | We can also use CPP flags that I proposed that would enable switching between
-- implementations without changing the code.
type WalletSscType = SscGodTossing
--type WalletSscType = SscNistBeacon
```